.github/workflows/pr-triage.yml

```diff
@@ -36,10 +34,10 @@ jobs:
     - name: Triages NEW issues and NEW pull requests to the Localization Project
```

Comment:
same as above, i think you can update the name for this now.s

Reply:
Updated!